ibis/backends/sqlite/tests/test_client.py

```diff
@@ -116,3 +116,55 @@ def test_verbose_log_queries(con):
     expected += 'FROM base.functional_alltypes AS t0\n'
     expected += ' LIMIT ? OFFSET ?'
     assert query == expected
+
+
+def test_insert_from_dataframe(con, df):
```

Comment:
This should be tested for all backends. Maybe you just need to find an `insert` test in `ibis/backends/tests` where the test is skipped for the Alchemy backends.

Reply:
I couldn't find any `insert` function in `ibis/backends/tests` directory which is being skipped. But I did find a few tests related to `load_data` and `create_table` etc but couldn't understand where it was being orchestrated (called) from. 
If you either find the `insert` test or if you still want me to assume the test function input (arguments) and write a test for `insert` then let me know. @datapythonista 

Thanks!

Reply:
This should be the file to test it: `ibis/backends/tests/test_client.py`

And you can use `@pytest.mark.only_on_backends(['sqlite', 'postgres', 'mysel'])`. And I think Impala and maybe other support this, so you can add them to the list too.

Reply:
Added the test to this file with `@pytest.mark.only_on_backends(['sqlite', 'postgres', 'mysql'], reason="run only if backend is SQLAlchemy based",)` since the backends based on SQLAlchemy have `insert` method signature different from others like `impala`.